JustSaying.AwsTools.IntegrationTests/project.json

```diff
@@ -1,57 +1,57 @@
-{
-  "version": "5.0.0",
-  "authors": [ "JUST EAT plc" ],
-  "copyright": "JUST EAT plc",
-  "title": "JustSaying",
-  "description": "A light-weight message bus on top of AWS SNS and SQS",
-  "buildOptions": {
-    "warningsAsErrors": true,
-    "platform": "anycpu",
-    "languageVersion": "csharp6",
-    "debugType": "portable"
-  },
-  "packOptions": {
-    "owners": [ "JUST EAT plc" ],
-    "summary": "SNS and SQS Messaging",
-    "tags": [ "aws", "sns", "sqs" ],
-    "repository": {
-      "type": "git",
-      "url": "https://github.com/justeat/JustSaying.git"
-    },
-    "requireLicenseAcceptance": true,
-    "licenseUrl": "https://github.com/justeat/JustSaying/blob/develop/LICENSE.md",
-    "projectUrl": "https://github.com/justeat/JustSaying"
-  },
-  "configurations": {
-    "Debug": {
-
-    },
-    "Release": {
-      "buildOptions": {
-        "optimize": true
-      }
-    }
-  },
-  "dependencies": {
-    "JustBehave": "1.0.0.12",
-    "JustSaying": "5.0.0",
-    "JustSaying.AwsTools": "5.0.0",
-    "JustSaying.Messaging": "5.0.0",
-    "JustSaying.Models": "5.0.0",
-    "JustSaying.TestingFramework": "5.0.0",
-    "Newtonsoft.Json": "9.0.1",
-    "NSubstitute": "1.10.0",
-    "NUnit": "3.6.0",
-    "AWSSDK.Core": "3.3.8",
-    "AWSSDK.SQS": "3.3.1.7",
-    "Microsoft.Extensions.Logging": "1.1.0",
-    "AWSSDK.SimpleNotificationService": "3.3.0.7",
-    "dotnet-test-nunit": "3.4.0-beta-3"
-  },
-  "testRunner": "nunit",
-  "frameworks": {
-    "net46": {
-      "dependencies": {}
-    }
-  }
-}
+{
+  "version": "5.0.*-beta01",

```

Comment:
I'd suggest using a bigger zero-pad (```0001```) just in case we get in a scenario where we just use the auto-generated build numbers and then it gets into the hundreds/thousands.